0001-fix-installation-paths-and-add-metainfo-file.patch

```diff
@@ -0,0 +1,143 @@
+From ead3a951dd39d5d1128c41cf4b34e9fbf99160f9 Mon Sep 17 00:00:00 2001
+From: Martin Marmsoler <martin.marmsoler@gmail.com>
+Date: Thu, 3 Nov 2022 17:01:53 +0100
+Subject: [PATCH] fix installation paths and add metainfo file
+
+---
+ source6/ChartGeany/ChartGeany.pro             | 13 +--
+ ...forge.chart_geany.chart-geany.metainfo.xml | 98 +++++++++++++++++++
+ 2 files changed, 103 insertions(+), 8 deletions(-)
+ create mode 100644 source6/ChartGeany/io.sourceforge.chart_geany.chart-geany.metainfo.xml
+
+diff --git a/source6/ChartGeany/ChartGeany.pro b/source6/ChartGeany/ChartGeany.pro
+index 39aad23..20ea0c0 100644
+--- a/source6/ChartGeany/ChartGeany.pro
++++ b/source6/ChartGeany/ChartGeany.pro
+@@ -129,15 +129,12 @@ QMAKE_CXXFLAGS += -fomit-frame-pointer -fstack-protector
+ # QMAKE_STRIP = echo
+ QMAKE_POST_LINK += strip $(TARGET)
+ 
+-target.path = /opt/$$PKG_NAME
+-images.path = /opt/$$PKG_NAME
+-desktop.path = /usr/share/applications/
+-images.files = images/$${PKG_NAME}.png
++target.path = $${PREFIX}/bin
++desktop.path = $${PREFIX}/share/applications/
+ desktop.files = $${PKG_NAME}.desktop
+-chartgeany.path = /opt/$$PKG_NAME
+-chartgeany.commands = rm -rf /usr/bin/$$TARGET; ln -s /opt/$$PKG_NAME/$$TARGET /usr/bin
+-chartgeany.uninstall = rm -rf /usr/bin/$$TARGET
+-INSTALLS += target desktop images chartgeany
++metainfo.path = $${PREFIX}/share/metainfo/
++metainfo.files = io.sourceforge.chart_geany.chart-geany.metainfo.xml
++INSTALLS += target desktop metainfo
+ }
+ 
+ #
+diff --git a/source6/ChartGeany/io.sourceforge.chart_geany.chart-geany.metainfo.xml b/source6/ChartGeany/io.sourceforge.chart_geany.chart-geany.metainfo.xml
+new file mode 100644
+index 0000000..89fba11
+--- /dev/null
++++ b/source6/ChartGeany/io.sourceforge.chart_geany.chart-geany.metainfo.xml
+@@ -0,0 +1,98 @@
++<?xml version="1.0" encoding="UTF-8"?>
++<component type="desktop-application">
++  <id>io.sourceforge.chart_geany.chart-geany</id>
++
++  <name>Chart Geany</name>
++  <summary>Chart Geany is a free cross platform software solution for market technical analysis, charting and a portfolio manager</summary>
++
++  <metadata_license>CC0-1.0</metadata_license>
++  <project_license>GPL-2.0-or-later</project_license>
++  <url type="homepage">https://chart-geany.sourceforge.io/</url>
++  <url type="help">https://sourceforge.net/p/chart-geany/discussion/</url>
++
++  <description>
++    <p>Chart Geany is a free cross platform software solution for market technical analysis, charting and a portfolio manager.</p>
```

Comment:
not really cross-platform given that it only support one hardware architecture.

Reply:
But it supports also windows and linux. I will get in contact with the author to see if we get it somehow to work on arm